Apps/FCM/ios/FCMSampleApp/MyAppPushNotificationsHandler.swift

```diff
@@ -1,36 +1,42 @@
+import Foundation
 import CioMessagingPushFCM
-import CioTracking
+import UserNotifications
 import FirebaseMessaging
-import Foundation
+import CioTracking
 
 @objc
-public class MyAppPushNotificationsHandler: NSObject {
+public class MyAppPushNotificationsHandler : NSObject {
 
   public override init() {}
 
+  @objc(setupCustomerIOClickHandling:)
+  public func setupCustomerIOClickHandling(withNotificationDelegate notificationDelegate: UNUserNotificationCenterDelegate) {
+    // This line of code is required in order for the Customer.io SDK to handle push notification click events.
+    // We are working on removing this requirement in a future release.
+    // Remember to modify the siteId and apiKey with your own values.
+    CustomerIO.initialize(siteId: Env.siteId, apiKey: Env.apiKey, region: Region.US) { config in
+      config.autoTrackDeviceAttributes = true
```

Comment:
Can we please set `config.logLevel = .debug` here for consistency when initializing the SDK in native classes?

Reply:
I do not think we should set any configurations here because it will override the configs set by the user using javascript initialisation method.

Reply:
> Can we please set config.logLevel = .debug here for consistency when initializing the SDK in native classes?

I chose not to do this because I did not include `config.logLevel` in the [SDK docs updates that I made](https://github.com/customerio/docs/pull/1447). 

I do agree that having `logLevel = .debug` would be really useful for our QA though. So, maybe...

```suggestion
      config.autoTrackDeviceAttributes = true
      
      // Configuration settings below are convenient for internal Customer.io testing. 
      // They are optional for your setup. 
      config.logLevel = .debug
```

?

Reply:
> I do not think we should set any configurations here because it will override the configs set by the user using javascript initialisation method.

I don't think it will override the javascript init method because of lifecycle. This line of code gets called before the javascript init method gets called. 

The config set here is because the native SDK init method used here is to make sure that `opened` metrics work. Therefore, we need this config set in order to assert `opened` metrics get reported before JS init method gets called.

Reply:
Yeah, that's what I had mainly in my mind when suggesting this, this is helpful for QA. Suggested change looks good to me 👍🏻